lxd/storage/backend_lxd.go

```diff
@@ -1798,6 +1798,33 @@ func (b *lxdBackend) DeleteImage(fingerprint string, op *operations.Operation) e
 	return nil
 }
 
+// UpdateImage updates image config.
+func (b *lxdBackend) UpdateImage(fingerprint, newDesc string, newConfig map[string]string, op *operations.Operation) error {
```

Comment:
This should fail if we try to alter any config though, unless you're planning on doing this validation at the api layer?

Reply:
@stgraber ok yes, I was silently ignoring the new config, but can fail instead.